src/fsae_firmware/magellan_controller/EncoderPublisher.h

```diff
@@ -0,0 +1,34 @@
+#ifndef ENCODER_PUBLISHER_H
+#define ENCODER_PUBLISHER_H
+
+// constants for quik maffs
+#define WHEEL_DIAMETER_IN 4.25          //inches
```

Comment:
Just a comment (feel free to keep as-is)

It would be more C++-like to have these as `constexpr` variables rather than `#define` macros